client/src/pages/Main.js

```diff
@@ -21,3 +45,37 @@ const ResultsText = styled.p`
   letter-spacing: 0.05rem;
   padding: 0.5rem;
 `;
+
+const ErrorWrapper = styled.section`
+  letter-spacing: 0.05rem;
+  padding: 4rem;
+  display: grid;
```

Comment:
In your structuring you usually have the display flex design and the associated adjustments in the first place :)

Reply:
thank you! :)